source/dsl/DangerUtilsDSL.ts

```diff
@@ -5,7 +5,7 @@
 export interface DangerUtilsDSL {
 
   /**
-   * Creates an HTML link.
+   * Creates a link using HTML.
```

Comment:
The previous one is not wrong, but if I already tried fixing it, I bet we'd get people in the future doing the same 👯‍♂️